### ISSUE TITLE
Issue with non-string properties?

### ISSUE THREAD

**Issue by mleech** [3factr/Serilog.Sink.AppCenter#4]
Hi,
Thanks for the very useful work on this sink.
I'm running into an issue where it looks like non-string (or at least integer) properties are causing events not to be sent to AppCenter. I haven't dug into it very far, but:
```
Serilog.Log.Information("Test Event {propFirst}; {propSecond}", "this is a test", 27);
```
doesn't make it to AppCenter, where-as:
```
Serilog.Log.Information("Test Event {propFirst}; {propSecond}", "this is a test", "27");
```
does. Note that the second property is now a string in the working version.

**Comment by FranVillaVainilla** [comment]
The problem seems to be in ConvertToProperties method in AppCenterSink class. If you change this line:

`property.Value.Render(stringWriter, "l", _formatProvider);`

with this:

`property.Value.Render(stringWriter, null, _formatProvider);`

... the problem disappears. That's mean changing format parameter from "l" to null solves the problem. I'm starting with App Center and Serilog just now, and i even does not understand what's the "l" format parameter trying to do, but the problem is there.

Note: I'm configuring AppCenterSink with a null IFormatProvider, does not know if this as something to do with the problem.

Thanks for the job @SamuelDebruyn

**Comment by sdebruyn** [comment]
Fixed in #15
